docs/demos.md

```diff
@@ -1,5 +1,17 @@
+# Some Demos
+
+## First Demo
+
 <demo src="./demo/demo-1.vue" attrs="{4}" title="Demo block - 1" desc="use demo" />
 
+## Second Demo
+
+<demo src="./demo/demo-2.vue" attrs="{4}" title="Demo block - 2" desc="use demo" />
+
 <!-- <demo src="./demo/demo-1.vue" attrs="{6}" title="Demo block - 2" v-bind="{a: 321}" desc="use `demo`" /> -->
 
-<!-- <demo src="./demo/demo-1.vue"  title="Demo block - 3" expand :a="123" /> -->
\ No newline at end of file
+<!-- <demo src="./demo/demo-1.vue"  title="Demo block - 3" expand :a="123" /> -->
+
+<script setup lang="ts">
```

Comment:
Having a <script setup> in an .md file that costumes the <demo> component give `[vitepress] Internal server error: Single file component can contain only one <script setup> element` - this is true even if the file only contains one component.